pkg/config/secret.go

```diff
@@ -64,8 +64,20 @@ func DecryptSecretHookFunc(hashKey string) mapstructure.DecodeHookFunc {
 		if t != reflect.TypeOf(result) {
 			return data, nil
 		}
+		// Get all env vars and filter for DOTENV_PRIVATE_KEY
+		var privateKeys []string
+		for _, env := range os.Environ() {
+			key := strings.Split(env, "=")[0]
+			if strings.HasPrefix(key, "DOTENV_PRIVATE_KEY") {
```

Comment:
I think this would also match `DOTENV_PRIVATE_KEYabc`. May be a regex match here would be nice.

Reply:
Used a direct match + prefix instead.